test/ae_database/AESetup.inc

```diff
@@ -0,0 +1,251 @@
+        $ae_str = "Enabled";
+    $connectionInfo = array("Database"=>$databaseName, "UID"=>$uid, "PWD"=>$pwd,
+                            "ReturnDatesAsStrings"=>true, "CharacterSet"=>"UTF-8", "ColumnEncryption"=>$ae_str);
+    $conn = sqlsrv_connect($server, $connectionInfo);
+    sqlsrv_assert($conn === false, "Could not connect.\n");
+    return ($conn);
+}
+
+function DropAETable($conn){
+    $sql_drop = "IF OBJECT_ID('dbo.test_AE', 'U') IS NOT NULL DROP TABLE [dbo].[test_AE]";
+    $stmt = sqlsrv_query($conn, $sql_drop);
+    sqlsrv_assert($stmt === false, "Error in dropping table.\n");
+}
+
+function CreateAETableTypes($conn, $types){
+    $encDet = "ENCRYPTED WITH (ENCRYPTION_TYPE = DETERMINISTIC, 
+               ALGORITHM = 'AEAD_AES_256_CBC_HMAC_SHA_256', 
+               COLUMN_ENCRYPTION_KEY = CEK1) NULL, ";
+    $encRand = "ENCRYPTED WITH (ENCRYPTION_TYPE = RANDOMIZED, 
+                ALGORITHM = 'AEAD_AES_256_CBC_HMAC_SHA_256', 
+                COLUMN_ENCRYPTION_KEY = CEK1) NULL";
+    $sql_create = "CREATE TABLE [dbo].[test_AE]([Id] [int] IDENTITY(1,1)";
+    
+    foreach ($types as $type) {
+        $sql_create .= ", ";
+        switch($type){
+            case "nvarchar":
+                $sql_create .= "[normNvarchar][nvarchar] (50) NOT NULL,
+                                [encDetNvarchar] [nvarchar] (50) COLLATE Latin1_General_BIN2
+                                $encDet
+                                [encRandNvarchar] [nvarchar] (50)
+                                $encRand";
+                break;
+            case "nvarcharmax":
+                $sql_create .= "[normNvarcharmax][nvarchar] (max) NOT NULL,
+                                [encDetNvarcharmax] [nvarchar] (max) COLLATE Latin1_General_BIN2
+                                $encDet
+                                [encRandNvarcharmax] [nvarchar] (max)
+                                $encRand";
+                break;
+            case "nchar":
+                $sql_create .= "[normNchar][nchar] (10) NULL,
+                                [encDetNchar] [nchar] (10) COLLATE Latin1_General_BIN2
+                                $encDet
+                                [encRandNchar] [nchar] (10)
+                                $encRand";
+                break;
+            case "char":
+                $sql_create .= "[normChar][char] (10) NULL,
+                                [encDetChar] [char] (10) COLLATE Latin1_General_BIN2
+                                $encDet
+                                [encRandChar] [char] (10)
+                                $encRand";
+                break;
+            case "varchar":
+                $sql_create .= "[normVarchar][varchar] (50) NULL,
+                                [encDetVarchar] [varchar] (50) COLLATE Latin1_General_BIN2
+                                $encDet
+                                [encRandVarchar] [varchar] (50)
+                                $encRand";
+                break;
```

Comment:
I was thinking of the same thing too. There is a way to do it, but it won't work for the data in AEData.inc anymore